docs/static/netflow-module.asciidoc

```diff
@@ -0,0 +1,157 @@
+[[netflow-module]]
+=== Logstash Netflow Module
+
+++++
+<titleabbrev>Netflow Module</titleabbrev>
+++++
+
+The Logstash Netflow module simplifies the collection, parsing, and
+visualization of network flow data. With a single command, the module parses
+network flow data, indexes the events into Elasticsearch, and installs a suite
+of Kibana dashboards to get you exploring your data immediately. 
+
+Logstash modules support Netflow Version 5 and 9.
+
+//REVIEWERS: Is the above statement true? What about IPFIX and sFlow?
+
+==== What is Flow Data?
+
+Flow data is information collected by network devices, such as routers and
+switches, that you can use to gain insight into the traffic flow and volume
+across your network. The flow data includes information about the source and
+destination IP addresses associated with each network connection, enabling
+you to accurately determine the client and server involved in each conversation.
+
```

Comment:
Looks good! Here's an edit to remove a couple of minor grammatical nits (dangling modifier and unclear antecedent). Also broke a few sentences into shorter sentences because editor-types keep telling me that shorter sentences are easier to read and translate.

"Netflow is a type of data record streamed from capable network devices. It contains information about connections traversing the device, and includes source IP addresses and ports, destination IP addresses and ports, types of service, VLANs, and other information that can be encoded into frame and protocol headers. With Netflow data, network operators can go beyond monitoring simply the volume of data crossing their networks. They can understand where the traffic originated, where it is going, and what services or applications it is part of."

Reply:
Super!